swift/ql/test/query-tests/Security/CWE-094/UnsafeJsEval.swift

```diff
@@ -184,7 +184,7 @@ func testUsage(_ sink: @escaping (String) async throws -> ()) {
 		let remoteData = Data(remoteString.utf8)
 
 		try! await sink(String(decoding: localData, as: UTF8.self)) // GOOD: the data is local
-		try! await sink(String(decoding: remoteData, as: UTF8.self)) // BAD: the data is remote
+		try! await sink(String(decoding: remoteData, as: UTF8.self)) // BAD [NOT DETECTED]: the data is remote (TODO: model Data taint sources)
```

Comment:
A model for `Data` is planned as well https://github.com/github/codeql-c-team/issues/1334 .  I think its another important one.

Reply:
... and now scheduled for next week.  Perfectly fine for this to be merged with a few missing results though.